frontend/kesaseteli/employer/sentry.server.config.js

```diff
@@ -0,0 +1,19 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
```

Comment:
Is this enough or should we wrap the server also with sentry? https://github.com/City-of-Helsinki/yjdh/blob/develop/frontend/shared/src/server/next-server.js#L27-L44

with this https://docs.sentry.io/platforms/javascript/guides/nextjs/:

```
import { withSentry } from "@sentry/nextjs";

const handler = async (req, res) => {
  res.status(200).json({ name: "John Doe" });
};

export default withSentry(handler);
```

Reply:
as we spoke, no need to do this